pkg/apis/wsecurity/v1alpha1/body.go

```diff
@@ -0,0 +1,182 @@
+package v1alpha1
+
+import "fmt"
+
+//////////////////// BodyProfile ////////////////
+
+// Exposes ValueProfile interface
+type BodyProfile struct {
+	Unstructured *SimpleValProfile  `json:"unstructured"`
+	Structured   *StructuredProfile `json:"structured"`
+}
+
+func (profile *BodyProfile) profileI(args ...interface{}) {
+	switch v := args[0].(type) {
+	case string:
+		profile.ProfileUnstructured(v)
+	default:
+		profile.ProfileStructured(v)
+	}
+}
+
+func (profile *BodyProfile) ProfileUnstructured(str string) {
+	profile.Structured = nil
+	profile.Unstructured = new(SimpleValProfile)
+	profile.Unstructured.Profile(str)
+}
+
+func (profile *BodyProfile) ProfileStructured(data interface{}) {
+	profile.Unstructured = nil
+	profile.Structured = new(StructuredProfile)
+	profile.Structured.Profile(data)
+}
+
+/*
+// Future: Missing implementation
```

Comment:
I'd remove these types of commented code for PRs and keep into a private branch

Reply:
I forgot to remove them after I added the implementation :)